toolkits/springernature/packages/springernature-header/scss/50-components/core.scss

```diff
@@ -33,19 +33,19 @@
 }
 
 .c-header__journal-title {
-	@include media-query('tablet-vertical') {
+	@include media-query('tablet-wide-width') {
 		font-size: 2.2rem;
 	}
 
-	@include media-query('tablet') {
+	@include media-query('desktop-width') {
 		font-size: 2.7rem;
 	}
 
 	font-family: $context--font-family-serif;
 	line-height: 4.6rem;
 	font-size: 2rem;
 	font-size: 1.5rem;
-	color: color('mineshaft');
+	color: color('black');
 
 	&.hide {
```

Comment:
this is the wrong naming convention, should be a modifier `c-header__journal-title--hide`

Reply:
It was there already but I can fix it sure

Reply:
Same as above but in regards to the `.hide` class to become `.c-header__journal-title--hide`. Maybe a job for @sky-jack in another PR. What do you think @alexkilgour ?

Reply:
I actually think in this case we should update the class. It's a simple change, and we are already making a breaking change to the component. If we don't then it's just another breaking change straight afterwards. Just change the class and detail in the HISTORY file that `hide` has changed to `xyz`